src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,5 @@
+package seedu.addressbook.data.person;
+
+public interface Printable {
+	String getPrintableString();
```

Comment:
You have not implemented the method getPrintableString(Printables... x) function. Try doing it too.